packages/trimerge-sync/src/merge-heads.ts

```diff
@@ -15,7 +15,10 @@ export type MergeCommitsFn = (
 
 export type SortRefsFn = (refA: string, refB: string) => number;
```

Comment:
from the other PR, IMO this interface should accept a `helpers` object. I think that would help avoid some of the acrobatics we need to do when constructing the trimerge client. It could be useful to think of it as a merge context.

Reply:
down to try that, want to make a separate PR?